src/core/etl/src/Flow/ETL/Row/EntryFactory.php

```diff
@@ -223,6 +182,14 @@ public function createAs(string $entryName, mixed $value, Definition|Type $defin
                 return string_entry($entryName, type_optional(type_string())->cast($value), $metadata);
             }
 
+            if ($type instanceof HTMLType) {
+                return string_entry($entryName, type_optional(type_string())->cast($value), $metadata);
+            }
+
+            if ($type instanceof NullType) {
```

Comment:
I believe this is already handled above so null type shouldn't reach here but if I'm wrong, please cover this with test

Reply:
In the case of processing rows that are always null, you will only pass when you specify the type; this can only be detected then. I added a test for this case.